module_6/tusk/index.js

```diff
@@ -0,0 +1,40 @@
+"use strict"
+
+// tusk #1
+const randomArr = [1, 2, "Ivan", "banana", "Kiev", "Table", 3, 4];
+
+function arrayCopy () {
```

Comment:
Малося на увазі, що функція повинна приймати масив як вхідний параметр.
```suggestion
function arrayCopy (array) {
```